ops/configurator/Dockerfile

```diff
@@ -1,8 +1,29 @@
-FROM frolvlad/alpine-oraclejdk8
-MAINTAINER Nikita Volkov
-COPY fabric-configurator-1.2.16.jar fabric-configurator-1.2.16.jar
+#steps to get OpenJDK8
+FROM alpine:3.9
+ENV LANG C.UTF-8
+RUN { \
+                echo '#!/bin/sh'; \
+                echo 'set -e'; \
+                echo; \
+                echo 'dirname "$(dirname "$(readlink -f "$(which javac || which java)")")"'; \
+        } > /usr/local/bin/docker-java-home \
+        && chmod +x /usr/local/bin/docker-java-home
+ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
+ENV PATH $PATH:/usr/lib/jvm/java-1.8-openjdk/jre/bin:/usr/lib/jvm/java-1.8-openjdk/bin
+ENV JAVA_VERSION 8u191
+ENV JAVA_ALPINE_VERSION 8.191.12-r0
+RUN set -x \
+        && apk add --no-cache \
+                openjdk8="$JAVA_ALPINE_VERSION" \
+        && [ "$JAVA_HOME" = "$(docker-java-home)" ]
+
+#Workaround to avoid exeptions formn netty
+RUN ln -s  /lib/libcrypto.so.1.1 /lib/libcrypt.so.1
+#fabric-skeleton specific configuration
+
+COPY fabric-configurator-fat-1.3.2.jar fabric-configurator-fat-1.3.2.jar
 COPY network network
 COPY chaincodes chaincodes
 COPY fabric.yaml fabric.yaml
-CMD java -cp fabric-configurator-1.2.16.jar com.luxoft.fabric.config.Configurator --type config
+CMD java -cp fabric-configurator-fat-1.3.2.jar com.luxoft.fabric.config.Configurator --type config
```

Comment:
Let's think of passing version from outside (using ARG or ENV)?

Reply:
I agree, moreover I  think we should not keep this jar in the source code repo, but download it from some binary repository instead.

This seems like improvement for me, while I think the first priority is to make the project working first and then polish it.